lib/commands/samples.ts

```diff
@@ -49,3 +49,39 @@ class CloneCommandParameter implements ICommandParameter {
 		}).future<boolean>()();
 	}
 }
+
+class PrintCordovaSamplesInformationCommand implements ICommand {
```

Comment:
Is it possible to create a generic command for this and eliminate the duplicated code with just passing the framework identifier and having 3 registrations for each framework?